examples/country-info/list-of-languages-by-name.graphql

```diff
@@ -0,0 +1,6 @@
+mutation ListOfLanguagesByName {
```

Comment:
Why this is a mutation?

Reply:
SOAP handler generates everything as a mutation because it is not possible to seperate them as queries and mutations.